tests/proptest.rs

```diff
@@ -134,8 +132,6 @@ fn timeout_resolve<DP: DependencyProvider>(
 }
 
 type NumVS = Range<u32>;
-#[cfg(feature = "serde")]
-type SemVS = Range<SemanticVersion>;
```

Comment:
I had a bunch of errors caused by that feature-gated import in the past, so now i inlined it